tensorflow/tools/android/inference_interface/asset_manager_filesystem.cc

```diff
@@ -240,8 +241,8 @@ string AssetManagerFileSystem::RemoveAssetPrefix(const string& name) {
   return string(piece);
 }
 
-bool AssetManagerFileSystem::DirectoryExists(
-    const std::string& fname /*, TransactionToken* token */) {
+bool AssetManagerFileSystem::DirectoryExists(const std::string& fname,
+                                             TransactionToken* token) {
```

Comment:
This shouldn't change, it's a local method of the class

Reply:
Yes, Sorry about that. Fixed